docs/source/host/server-setup/linux-server.md

```diff
@@ -0,0 +1,91 @@
+
+Connect a web browser to the server at port 40100 (if you are on the machine, this would be http://localhost:40100). Complete the initial setup wizard and you should have a functioning domain.
+
+## Networking
+
+A Project Athena domain reserves a range of four ports to operate on, usually starting at 40100 (Note that the encrypted ports may not be implemented yet). These ports are:
+ - 40100 (+0) : (tcp) administrative http connection
+ - <del>40101 (+1): (tcp) administrative https (encrypted) connection</dev>
+ - 40102 (+2): (udp) main connection from clients
+ - <del>40103 (+3): (udp) main connection from clients (encrypted)</dev>
+
+Generally speaking only port 40102 must be publicly exposed to permit others to connect to a domain.
+
+In addition there are six "assignment clients" that must run in order for the domain to be operational.  These clients run alongside the domain server and must be able to connect with any outside user.  These clients are:
+- **asset-server**: stores object data to download to the user
+- **audio-mixer**: controls what each user would hear
+- **avatar-mixer**: allows users to see each other
+- **entity-script-server**: runs actions inside the domain
+- **entity-server**: describes the location of each object in the domain
+- **messages-mixer**: passes messages between users in the domain
+
+These assignment clients use UDP connections on a port number assigned by the operating system at launch (within [the ephemeral port range](https://en.wikipedia.org/wiki/Ephemeral_port)).
+
+## Files
+
+The program files are installed in /opt/athena:
+ - **/opt/athena** contains the executables
+ - **/opt/athena/lib** contains libraries required for operation
+ - **/opt/athena/plugins** is currently used for audio codecs
+ - **/opt/athena/resources** is required for the administrative website
+
+The executables in this folder (with the exception of <code>new-server</code>) cannot be launched from the command prompt without first setting <code>LD_LIBRARY_PATH=/opt/athena/lib</code>.
+
+The file <code>/etc/opt/athena/default.conf</code> contains any environment variables necessary to running the domain.
+
+All content is stored under <code>/var/lib/athena/default</code>.  All files underneath <code>/var/lib/athena</code> are owned by the user <code>athena</code>, which is also the user that runs all domain-related processes.
+
+## Services
+
+The installation packages setup the following systemd services to manage the Athena domain:
+- **athena-domain-server@default.service**: Manages the core domain server
+- **athena-assignment-client@default.service**: Spawns and manages the assignment clients
+- **athena-server@default.target**: Controls startup and shutdown of the above services
+
+The <code>athena-server@default.target</code> service is the only one that is set to auto-start, starting or stopping it will bring the other two services down.
```

Comment:
```suggestion
The <code>athena-server@default.target</code> service is the only one that is set to auto-start. Starting or stopping it will bring the other two services down.
```